clickhouse_std.go

```diff
@@ -272,8 +293,12 @@ func (std *nativeTransport) PrepareContext(ctx context.Context, query string) (d
 
 func (std *nativeTransport) Close() error { return std.conn.close() }
 
+type IBatch interface {
+	Append(v ...interface{}) error
+}
+
 type stdBatch struct {
-	batch *batch
+	batch IBatch
```

Comment:
```suggestion
	batch driver.Batch
```
Here is can be `Batch` interface from `lib` directory.

If that's a better solution.

Reply:
yes i think so